accelerator/cpu_accelerator.py

```diff
@@ -48,9 +48,11 @@ def device_count(self):
             # Ignore these NUMA nodes with no cores.
             numa_core_lists = get_numa_cores()
             numa_count = 0
+            prev_core_list = []
             for core_list in numa_core_lists:
-                if len(core_list) > 0:
+                if len(core_list) > 0 and core_list != prev_core_list:
```

Comment:
I assume this is to avoid duplicate core lists. If so, will the detection work if the duplicates are not consecutive in `numa_core_lists`. For example, would it detect the `[0,1]` duplicates if the `num_core_lists` was: 

```python
  [[0, 1] [2, 3] [0, 1]]
```

Reply:
Yes, this assume consecutive duplications.  Let me read fakenuma document to understand whether interleaving is allowed.

Reply:
From this document (5.3.1), fakenuma nodes is configured from kernel options.
https://www.intel.com/content/www/us/en/content-details/769060/intel-xeon-cpu-max-series-configuration-and-tuning-guide.html

From what I understand, the memory of a numa node is divided into several consecutive fakenuma node, and the CPU cores belong to the original numa node will be assigned to all the fakenuma node created from the original numa node, causing core list duplication.  So core list duplications should be consecutive.